components/suggestion/at_mention_provider/at_mention_provider.test.tsx

```diff
@@ -49,12 +52,12 @@ describe('components/suggestion/at_mention_provider/AtMentionProvider', () => {
         const itemsCall3 = [
             {type: Constants.MENTION_MEMBERS, ...userid10},
             {type: Constants.MENTION_MEMBERS, ...userid3},
-            {type: Constants.MENTION_MEMBERS, ...userid1},
+            {...userid1, type: Constants.MENTION_MEMBERS},
             {type: Constants.MENTION_MEMBERS, ...userid2},
             {type: Constants.MENTION_MEMBERS, ...userid4},
-            {type: Constants.MENTION_GROUPS, ...groupid1},
-            {type: Constants.MENTION_GROUPS, ...groupid2},
-            {type: Constants.MENTION_GROUPS, ...groupid3},
+            {...groupid1, type: Constants.MENTION_GROUPS},
+            {...groupid2, type: Constants.MENTION_GROUPS},
+            {...groupid3, type: Constants.MENTION_GROUPS},
```

Comment:
I changed this since you will get this error when you change the file to typescript
https://github.com/microsoft/TypeScript/issues/38535

```
'type' is specified more than once, so this usage will be overwritten.ts(2783)
at_mention_provider.test.tsx(57, 46): This spread always overwrites this property.
```